lib/utils/location.js

```diff
@@ -51,12 +53,14 @@ export const execRegexes: { [string]: (path: string) => ?string[] } = {
 	},
 };
 
-export type PageType = 'wiki' | 'search' | 'profile' | 'profile2x' | 'profileCommentsPage' | 'stylesheet' | 'modqueue' | 'subredditAbout' | 'comments' | 'commentsLinklist' | 'liveThread' | 'inbox' | 'submit' | 'account' | 'prefs' | 'linklist';
-export type AppType = 'r2' | 'd2x';
+export type PageType = 'wiki' | 'search' | 'profile' | 'profile2x' | 'profileCommentsPage' | 'stylesheet' | 'modqueue' | 'subredditAbout' | 'comments' | 'commentsLinklist' | 'liveThread' | 'inbox' | 'submit' | 'account' | 'prefs' | 'linklist' | 'newModMail' | 'composeNewModMail';
+export type AppType = 'r2' | 'd2x' | 'nmm';
```

Comment:
How about `mm2x`?  Internally at Reddit, "new modmail / modmail beta" is part of the "2x" family, i.e. Node-served React webapps.